debian/control

```diff
@@ -0,0 +1,54 @@
+Source: ddfacet
+Section: science
+Priority: extra
+Maintainer: Debian Astro Team <debian-astro-maintainers@lists.alioth.debian.org>
+Build-Depends: cmake,
+               gfortran,
+               python-setuptools,
+               dh-python,
+               python-all,
+               python-dev,
+               pkg-config,
+               debhelper (>=9),
+               python-nose (>=1.3.7),
+               cython (>=0.25.2),
+               python-numpy (>=1.11.0),
+               python-sharedarray (>=2.0.2),
+               polygon2 (>=2.0.8),
+               python-pyfftw (>=0.10.4),
+               python-astropy (>=1.3.3),
+               python-deap (>=1.0.1),
+               python-ipdb (>=0.10.3),
+               python-casacore (>=2.1.0),
+               python-ephem (>=3.7.6.0),
+               python-numexpr (>=2.6.2),
+               python-matplotlib (>=2.0.0),
+               python-scipy (>=0.16.0),
+               python-astro-kittens (>=0.3.3),
+               meqtrees-cattery (>=1.5.1),
```

Comment:
that should be `python-meqtrees-cattery`. It is a python only library and Debian requires to have a `python-` prefix for those.

Reply:
Duly